packages/theme-language-server-common/src/hover/providers/RenderSnippetHoverProvider.spec.ts

```diff
@@ -16,57 +16,42 @@ describe('Module: RenderSnippetHoverProvider', async () => {
           name: 'title',
           description: 'The title of the product',
           type: 'string',
+          required: true,
         },
         {
           name: 'border-radius',
           description: 'The border radius in px',
           type: 'number',
+          required: false,
         },
         {
           name: 'no-type',
           description: 'This parameter has no type',
           type: null,
+          required: true,
         },
         {
           name: 'no-description',
           description: null,
           type: 'string',
+          required: true,
         },
         {
           name: 'no-type-or-description',
           description: null,
           type: null,
+          required: true,
         },
       ],
     },
   };
 
-  const createProvider = (getSnippetDefinition: GetSnippetDefinitionForURI) => {
-    return new HoverProvider(
-      new DocumentManager(),
-      {
-        filters: async () => [],
-        objects: async () => [],
-        tags: async () => [],
-        systemTranslations: async () => ({}),
-      },
-      async (_rootUri: string) => ({} as MetafieldDefinitionMap),
-      async () => ({}),
-      async () => [],
-      getSnippetDefinition,
-    );
-  };
-
-  beforeEach(async () => {
```

Comment:
I want to make extending custom cases easier, rather than having a massive test case that's hard to read / understand